example-meal-order/src/app/extensions/components/OrderMealCard.tsx

```diff
@@ -116,10 +118,10 @@ export const OrderMealCard = ({
 
   // Small utility function for help below
   const getRestaurant = (id?: number) => {
-    return restaurants.find(r => r.id === id);
+    return restaurants.find((r) => r.id === id);
```

Comment:
I'm guessing this is starting to apply the prettier default config since we didn't add one explicitly?

Reply:
Hmm, maybe my VSCode did this automatically. I assumed it was part of the linting. Let me turn that off.

Reply:
Oh wait, it must be using the [top level config](https://github.com/HubSpot/ui-extensions-examples/blob/main/.prettierrc.json) cuz I have this setting set in VSCode.

<img width="395" alt="image" src="https://github.com/HubSpot/ui-extensions-examples/assets/134086585/8d5616ab-f8bc-423d-8d19-d4e02f5dc2a0">